### PR TITLE
Fix badge url

### PR DESCRIPTION
**Pull request by PunitLodha**
Fixed the badge urls in Contributing.md

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -125,8 +125,8 @@ There are a lot of open issues on the backlog for those interested in hacking on
 
 Here are some good starting issues:
 
-[![GitHub good-first-issue](https://img.shields.io/github/issues/rust-lang/rustfmt/good%20first%20issue?style=flat-square)](https://github.com/rust-lang/rustfmt/issues?q=is%3Aopen+is%3Aissue+label%3Agood%20first%20issue)
-[![GitHub help-wanted](https://img.shields.io/github/issues/rust-lang/rustfmt/help%20wanted?style=flat-square)](https://github.com/rust-lang/rustfmt/issues?q=is%3Aopen+is%3Aissue+label%3Ahelp-wanted)
+[![GitHub good-first-issue](https://img.shields.io/github/issues/rust-lang/rustfmt/good%20first%20issue?style=flat-square)](https://github.com/rust-lang/rustfmt/issues?q=is%3Aopen+is%3Aissue+label%3A\"good%20first%20issue\")
```
Comment (calebcartwright) [inline Contributing.md:128]:
```suggestion
[![GitHub good-first-issue](https://img.shields.io/github/issues/rust-lang/rustfmt/good%20first%20issue?style=flat-square)](https://github.com/rust-lang/rustfmt/issues?q=is%3Aopen+is%3Aissue+label%3A%22good%20first%20issue%22)
```

File: Contributing.md [hunk 2]
```diff
@@ -125,8 +125,8 @@ There are a lot of open issues on the backlog for those interested in hacking on
...
+[![GitHub help-wanted](https://img.shields.io/github/issues/rust-lang/rustfmt/help%20wanted?style=flat-square)](https://github.com/rust-lang/rustfmt/issues?q=is%3Aopen+is%3Aissue+label%3A\"help%20wanted\")
```
Comment (calebcartwright) [inline Contributing.md:129]:
```suggestion
[![GitHub help-wanted](https://img.shields.io/github/issues/rust-lang/rustfmt/help%20wanted?style=flat-square)](https://github.com/rust-lang/rustfmt/issues?q=is%3Aopen+is%3Aissue+label%3A%22help%20wanted%22)
```

**Review by calebcartwright — COMMENTED** [review]
Thanks for spotting this and sending a PR! Would really prefer to use uri encoding instead of escaped quotes here so inline changes suggested to that end, otherwise LGTM

**Comment by PunitLodha** [comment]
Ok. I will make these changes

**Comment by calebcartwright** [comment]
Thanks!

**Pull request merged**